### ISSUE TITLE
Stack-view focus outline not visible in Safari

### ISSUE THREAD

**Issue by Jinnie** [vmware-clarity/ng-clarity#31]
**This is a bug report for the `@clr` Angular or UI versions of the design system.
For the web-component implementation of Clarity (`@cds`), visit [vmware-clarity/core](https://github.com/vmware-clarity/core).**

## Describe the bug

The focus outline, which can be observed on Chrome is not visible in Safari
## How to reproduce

1.  Go to https://angular.clarity.design/documentation/stack-view using Safari browser
2.  Tab until you have to land on an expandable node
3.  No focus outline can be seen. You can still hit enter to expand/collapse, to verify the focus is actually there.
